src/llvm-final-gc-lowering.cpp

```diff
@@ -108,20 +108,20 @@ void FinalLowerGC::lowerPushGCFrame(CallInst *target, Function &F)
                 ConstantInt::get(T_size, JL_GC_ENCODE_PUSHARGS(nRoots)),
                 builder.CreateBitCast(
                         builder.CreateConstInBoundsGEP1_32(T_prjlvalue, gcframe, 0),
-                        T_size->getPointerTo()),
+                        T_size->getPointerTo(), "nroots"),
                 Align(sizeof(void*)));
     inst->setMetadata(LLVMContext::MD_tbaa, tbaa_gcframe);
     auto T_ppjlvalue = JuliaType::get_ppjlvalue_ty(F.getContext());
     inst = builder.CreateAlignedStore(
-            builder.CreateAlignedLoad(T_ppjlvalue, pgcstack, Align(sizeof(void*))),
+            builder.CreateAlignedLoad(T_ppjlvalue, pgcstack, Align(sizeof(void*)), "pgcstack_frame"),
             builder.CreatePointerCast(
                     builder.CreateConstInBoundsGEP1_32(T_prjlvalue, gcframe, 1),
-                    PointerType::get(T_ppjlvalue, 0)),
+                    PointerType::get(T_ppjlvalue, 0), "new_frame"),
             Align(sizeof(void*)));
     inst->setMetadata(LLVMContext::MD_tbaa, tbaa_gcframe);
     inst = builder.CreateAlignedStore(
             gcframe,
-            builder.CreateBitCast(pgcstack, PointerType::get(PointerType::get(T_prjlvalue, 0), 0)),
+            builder.CreateBitCast(pgcstack, PointerType::get(PointerType::get(T_prjlvalue, 0), 0), "pgcstack_frame"),
```

Comment:
I personally don't see much point in naming this bitcast the same as the `pgcstack_frame` load above, plus once we migrate to opaque pointers the bitcast will vanish.

Reply:
I guess we should just name `pgcstack` instead.